persistence/src/main/scala/hmda/persistence/processing/SubmissionFSM.scala

```diff
@@ -230,6 +230,15 @@ class SubmissionFSM(submissionId: SubmissionId)(implicit val domainEventClassTag
       val status = hmda.model.fi.Signed
```

Comment:
I don't know, it's a good question. I believe the frontend might prevent this scenario from happening, but not sure. In any case, if this change is needed I would like to do it in a different PR.